tests/test_mysensors.py

```diff
@@ -462,15 +462,6 @@ def test_non_presented_child(protocol_version, return_value):
     assert ret == return_value
 
 
-def test_set_child_no_children(gateway, add_sensor):
```

Comment:
Why do we remove this test?

Reply:
@MartinHjelmare it was testing behavior if we pass the empty children array to the set method.
This parameter is removed, so there is no such case now.
I've extended the previous test with the set value attempt (I think it was missing there)